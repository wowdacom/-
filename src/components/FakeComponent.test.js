import { expect, test } from "vitest";
import { mount } from "@vue/test-utils";
import component from "./FakeComponent.vue";
import { nextTick } from "vue";

test("snapshot", async () => {
  const wrapper = mount(component);
  await nextTick();
  expect(wrapper.find('[data-test="target"]').html()).toMatchSnapshot();
});
