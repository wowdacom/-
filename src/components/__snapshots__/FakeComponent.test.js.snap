// Vitest Snapshot v1

exports[`snapshot 1`] = `
"<h1 data-test=\\"target\\" id=\\"header\\" data-v-a7e46a08=\\"\\">
  <p>公館陳柏霖 您好：
    本次總票價為 <span style=\\"text-decoration:line-through\\">2,550$</span></p><br>
  <p>實際消費為 2,155$</p>
  <p>此次會員資格為您節省了 395$</p>
</h1>"
`;

exports[`snapshot 2`] = `
"<h1 data-test=\\"target\\" id=\\"header\\" data-v-a7e46a08=\\"\\">
  <p>公館陳柏霖 您好：
    本次總票價為 <span style=\\"text-decoration:line-through\\">2,550$</span></p><br>
  <p>實際消費為 2,155$</p>
  <p>此次會員資格為您節省了 395$</p>
</h1>"
`;
